### DISCUSSION TITLE
Work on OS X Sonoma yet? Argyll version should I use?

### DISCUSSION THREAD

**Discussion by nquantum** [eoyilmaz/displaycal-py3#301] [Q&A]
Does this version Displaycal work on OS X Sonoma yet? I have MacBook Pro M1'client with Sonoma. Previous Displaycal failed to start andI found fork version here. Also what version of Argyll should I work with M1? 3.0.1?

Thank.

**Comment by MStraeten** [comment]
Main branch installed via https://github.com/eoyilmaz/displaycal-py3#how-to-install works fine on a m1max with sonoma (using macports instead of homebrew)
If you own a xrite or Calibrite device you need 3.0.2 (or 2.3.1) but you need to download and install it manually

**Comment by MStraeten** [comment]
for the brave ones: a macports based build for arm64:
[Argyll_V3.1.0.zip](https://mega.nz/file/DHYyQCgS#eCE6wLd7ihla4mXKYjBBJCyVyuiefZR8sOTIW9wc1lw)
binaries are unsigned so you you need to convince your mac that this stuff is ok :wink:
`xattr -dr com.apple.quarantine * <wherever you unpack that zip file>/Argyll_V3.1.0/bin`

**Comment by 384842** [comment]
I try use DisplayCAL-3.9.14-macOS_arm64 today on macOS 15 Sequoia, M2 Pro, but it not start properly.

**Comment by gian2k** [comment]
The DisplayCAL-3.9.14-macOS_arm64 starts and asks for screen recording every time it starts and cannot find the probe, Tested witch Mac mini m4 256gb.

**Comment by MStraeten** [comment]
that’s annoying but doesn’t block calibration and profiling. macOS 15 increased security demands, so you might need to allow execution for the argyll binaries manually

**Comment by MigCurto** [comment]
Working fine (as oposed to "offical" download) on iMac 2019 with Sonoma 14.6.1
I did have to use the DisplayCAL-3.9.14-py3-none-any.whl 

thank you sir.
